broker/logger.py

```diff
@@ -20,8 +20,39 @@ class LOG_LEVEL(IntEnum):
     ERROR = logging.ERROR
 
 
+class RedactingFilter(logging.Filter):
+    """Custom logging.Filter to redact secrets from the Dynaconf config"""
+    def __init__(self, sensitive):
+        super(RedactingFilter, self).__init__()
+        self._sensitive = sensitive
+
+    def filter(self, record):
+        if isinstance(record.args, dict):
+            record.args = self.redact_dynaconf(record.args)
+        else:
+            record.args = tuple(self.redact_dynaconf(arg) for arg in record.args)
+        return True
+
+    def redact_dynaconf(self, data):
+        """
+        This method goes over the data and redacts all values of keys
+        that match the sensitive ones
+        """
+        if isinstance(data, (list, tuple)):
+            data_copy = [self.redact_dynaconf(item) for item in data]
+        elif isinstance(data, dict):
+            data_copy = copy.deepcopy(data)
+            for k, v in data_copy.items():
+                if isinstance(v, (dict, list)):
+                    data_copy[k] = self.redact_dynaconf(v)
+                elif k in self._sensitive and v:
+                    data_copy[k] = "*" * 6
```

Comment:
Much easier to just write out the string
```suggestion
                    data_copy[k] = "******"
```